internal/auth0/customdomain/data_source_test.go

```diff
@@ -11,15 +11,56 @@ import (
 	"github.com/auth0/terraform-provider-auth0/internal/acctest"
 )
 
-const testAccDataSourceCustomDomain = `
+const testAccDataSourceCustomDomainSingle = `
 resource "auth0_custom_domain" "my_custom_domain" {
-	domain     = "{{.testName}}.auth.terraform-provider-auth0.com"
+	domain     = "{{.testName}}.auth.tempdomain.com"
 	type       = "auth0_managed_certs"
 	tls_policy = "recommended"
+	domain_metadata = {
+        key1: "value1"
+		key2: "value2"
+    }
 }
 
 data "auth0_custom_domain" "test" {
-	depends_on = [ auth0_custom_domain.my_custom_domain ]
+	depends_on = [resource.auth0_custom_domain.my_custom_domain]
```

Comment:
```suggestion
	depends_on = [ auth0_custom_domain.my_custom_domain ]
```

we should revert it back to `	depends_on = [ auth0_custom_domain.my_custom_domain ]`

as The prefix resource. does NOT exist in Terraform interpolation or references.

`<resource_type>.<resource_name>`